Chapter-1-initial-architecture/Src/Fitnet.ArchitectureTests/Contracts/ContractsModuleArchitectureTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.ArchitectureTests.Contracts;
+
+using System.Reflection;
+using Common.Predicates;
+using NetArchTest.Rules;
+
+public class ContractsModuleArchitectureTests
```

Comment:
I am lacking other tests like:

- ShouldNotHaveDependencyOnXModule
```
 [Fact]
    internal void ShouldNotHaveDependencyOnXModule()
    {
        // Act
        var result = Types.That()
            .ResideInNamespace(ContractNamespace)
            .ShouldNot()
            .HaveDependencyOn(Xnamespace)
            .GetResult();

        // Assert
        result.IsSuccessful.Should().BeTrue();
    }

    [Fact]
    internal void ShouldNotHaveDependencyOnApi_WhenInsideData()
    {
        // Act
        var result = Types.That()
            .ResideInNamespace(DataNamespace)
            .ShouldNot()
            .HaveDependencyOn(ApiNamespace)
            .GetResult();

        // Assert
        result.IsSuccessful.Should().BeTrue();
    }

    [Fact]
    internal void ShouldBeStatic_WhenClassIsEndpoint()
    {
        // Act
        var result = Types.That()
            .AreClasses()
            .And()
            .HaveNameEndingWith("Endpoint")
            .Should()
            .BeStatic()
            .GetResult();

        // Assert
        result.IsSuccessful.Should().BeTrue();
    }
    ```
    etc. Will you add it in the next branch?

Reply:
yeah convention tests will be in the future.